eZ/Publish/Core/settings/search_engines/field_value_mappers.yml

```diff
@@ -2,72 +2,74 @@ services:
     ezpublish.search.common.field_value_mapper.boolean:
         class: eZ\Publish\Core\Search\Common\FieldValueMapper\BooleanMapper
         tags:
-            - {name: ezpublish.search.common.field_value_mapper}
+            - { name: ezpublish.search.common.field_value_mapper, maps: eZ\Publish\SPI\Search\FieldType\BooleanField }
```

Comment:
Remember when merging this up, that this tag has to be repeated for aliased classes.